content/components/selectpanel.mdx

```diff
@@ -190,35 +203,9 @@ By employing these loading states, users can have a better understanding of the
   </Dont>
 </DoDontContainer>
 
-### Click-to-dismiss behavior
-
```

Comment:
Instead of removing this section completely, should we replace this with the behavior we want now?